Makefile

```diff
@@ -46,4 +49,17 @@ ineffassign:
 run: build
 	$(shell bin/vegeta-server --scheme=http --host=localhost --port=8000)
 
+
+container:
+	docker build -t vegeta-server:latest .
+
+container_run:
+	docker build -t vegeta-server:latest .
+	@docker rm -f vegeta || true
+	@docker run -d -p 8000:80 --name vegeta vegeta-server:latest
```

Comment:
Add `--rm` flag to `docker run`. This will ensure that the container gets removed, if it happens to crash or exit. It will remove containers that drop to `status: exited`

Reply:
Thanks I didn't know about that!